Module_Postman/activities/Act5_Mock_Servers/3.md

```diff
@@ -0,0 +1,17 @@
+<!--title={Generating Examples}-->
+
+After setting up your mock server, you will start working in Postman Echo!
+
+The **Postman Echo** service allows users to experiment with various responses and, first and foremost, can be used to generate example responses, which we will focus on in this lesson. In order to mock requests properly, we need to know how to generate examples for the mock server to "learn" from. 
+
+![using-echo-1](using-echo-1.png)
+
+To continue from where we left off from the last lesson, go to the upper right-hand side of your window to click on "Examples" and choose "Default". You can name your example and change your headers, body, status code, and example response body, which is specifically what we will be changing in the photo below.
```

Comment:
This can be a tricky part of the Postman ui to find, so I'd maybe just spell it out a little more in case people get lost in between the last section and this one e.g. "with the request you sent to your mock server open..." or something